src/Core/Grid/Definition/Factory/Security/Sessions/EmployeeGridDefinitionFactory.php

```diff
@@ -0,0 +1,273 @@
+            ->add(
+                (new Filter('id_employee_session', TextType::class))
+                    ->setTypeOptions([
+                        'required' => false,
+                        'size' => 'small',
+                        'attr' => [
+                            'placeholder' => $this->trans('Search ID', [], 'Admin.Actions'),
+                        ],
+                    ])
+                    ->setAssociatedColumn('id_employee_session')
+            )
+            ->add(
+                (new Filter('id_employee', TextType::class))
+                    ->setTypeOptions([
+                        'required' => false,
+                        'size' => 'small',
+                        'attr' => [
+                            'placeholder' => $this->trans('Search ID', [], 'Admin.Actions'),
+                        ],
+                    ])
+                    ->setAssociatedColumn('id_employee')
+            )
+            ->add(
+                (new Filter('firstname', TextType::class))
+                    ->setTypeOptions([
+                        'required' => false,
+                        'attr' => [
+                            'placeholder' => $this->trans('First name', [], 'Admin.Global'),
+                        ],
+                    ])
+                    ->setAssociatedColumn('firstname')
+            )
+            ->add(
+                (new Filter('lastname', TextType::class))
+                    ->setTypeOptions([
+                        'required' => false,
+                        'attr' => [
+                            'placeholder' => $this->trans('Last name', [], 'Admin.Global'),
+                        ],
+                    ])
+                    ->setAssociatedColumn('lastname')
+            )
+            ->add(
+                (new Filter('email', TextType::class))
+                    ->setTypeOptions([
+                        'required' => false,
+                        'attr' => [
+                            'placeholder' => $this->trans('Email', [], 'Admin.Global'),
+                        ],
+                    ])
+                    ->setAssociatedColumn('email')
+            )
+            ->add(
+                (new Filter('actions', SearchAndResetType::class))
+                    ->setTypeOptions([
+                        'attr' => [
+                            'data-url' => $this->resetActionUrl,
+                            'data-redirect' => $this->redirectionUrl,
+                        ],
+                    ])
+                    ->setAssociatedColumn('actions')
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getBulkActions(): BulkActionCollectionInterface
+    {
+        return (new BulkActionCollection())
+            ->add(
+                (new SubmitBulkAction('bulk_delete_sessions_employees'))
+                    ->setName($this->trans('Delete selected', [], 'Admin.Actions'))
```

Comment:
```suggestion
                    ->setName($this->trans('Delete selection', [], 'Admin.Actions'))
```

One last that I missed